server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/tornado/utils/TornadoCalc.java

```diff
@@ -11,6 +11,7 @@
 
 import com.google.common.math.LongMath;
 import edu.illinois.ncsa.incore.service.hazard.models.eq.utils.HazardUtil;
+import edu.illinois.ncsa.incore.service.hazard.models.flood.FloodHazardDataset;
```

Comment:
Oh why is flood needed here?

Reply:
good catch! it isn't its something IntelliJ imported when I copy pasted a function